src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer start working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Stop start working!");
```

Comment:
The message in the `stopWork` method is not informative about the type of Machine that stopped working. According to the checklist, it should indicate the specific type of machine, in this case, `Bulldozer`. Consider changing the message to something like 'Bulldozer stopped working!' to make it clear and informative.